ballerina/modules/charitycheckpdf/client.bal

```diff
@@ -1,7 +1,4 @@
-// AUTO-GENERATED FILE. DO NOT MODIFY.
-// This file is auto-generated by the Ballerina OpenAPI tool.
```

Comment:
We need to keep this "AUTO-GENERATED" header part in all the generated files

Reply:
This is getting removed when we generate, I think I might have just copied the license header last time, when we generate with the license header, the AUTO-GENERATED thing is getting removed and the license header is placed instead.

Reply:
@AzeemMuzammil yes thats the expected behaviour from the openAPI tool IINM (@lnash94 please confirm). 

As a solution for that we have to add the auto generated header part on top of the wso2 license text template file we keep in our resources

Reply:
I see, will do that,